README.md

```diff
@@ -376,37 +418,173 @@ imagekitio.update_file_details(file_id, {
+  temp_file.write(chunk)
+end
+```
+
+**13. Add Bulk Tags**
+
+Add multiple tags on multiple files using array of file ids and array of tags as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/add-tags-bulk)
+
+```ruby
+imagekitio.add_bulk_tags(['id_1', 'id_2'], ['custom_tags', 'image', 'favourite'])
+``` 
+
+**14. Remove Bulk Tags**
+
+Remove multiple tags from multiple files using array of file ids and array of tags as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/remove-tags-bulk)
+
+```ruby
+imagekitio.remove_bulk_tags(['id_1', 'id_2'], ['custom_tags', 'image'])
+```
+
+**15. Create Folder**
+
+Create folder as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/create-folder)
+
+```ruby
+imagekitio.create_folder('new_folder', 'source/folder/path')
+```
+
+
+**16. Copy Folder**
+
+Copy folder as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/copy-folder)
+
+```ruby
+imagekitio.copy_folder('/folder/to/copy', '/folder/to/copy/into')
+```
+
+**17. Move Folder**
+
+Move folder as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/move-folder)
+
+```ruby
+imagekitio.move_folder('/folder/to/move', '/folder/to/move/into/')
+```
+
+**18. Delete Folder**
+
+Delete folder as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-folder)
+
+```ruby
+imagekitio.delete_folder('folder/to/delete')
+```
+
+**19. Bulk Job Status**
+
+Get the bulk job status as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/copy-move-folder-status)
+
+```ruby
+imagekitio.bulk_job_status('job_id')
+```
+
+**20. Create Custom Metadata Fields**
+
+Create custom metadata fields as per the [API documentation here](https://docs.imagekit.io/api-reference/custom-metadata-fields-api/create-custom-metadata-field)
+
+```ruby
+
+imagekitio.create_custom_metadata_fields(
+    'price', 
+    'price_label', 
+    {
+      type: 'Number',
+      minValue: 100,
+      maxValue: 300
+    }
+)
+```
+
+**21. Get Custom Metadata Fields**
+
+Get the custom metadata fields as per the [API documentation here](https://docs.imagekit.io/api-reference/custom-metadata-fields-api/get-custom-metadata-field)
+
+```ruby
+imagekitio.get_custom_metadata_fields
+```
+
+**22. Update Custom Metadata Fields**
+
+Update custom metadata fields as per the [API documentation here](https://docs.imagekit.io/api-reference/custom-metadata-fields-api/update-custom-metadata-field)
+
+```ruby
+imagekitio.update_custom_metadata_fields('file_id', label: 'custom-price', schema: nil)
+```
+
+**23. Delete Custom Metadata Fields**
+
+Delete custom metadata fields as per the [API documentation here](https://docs.imagekit.io/api-reference/custom-metadata-fields-api/delete-custom-metadata-field)
+
+```ruby
+imagekitio.delete_custom_metadata_fields('file_id')
```

Comment:
`file_id` seems to be incorrect. Let's keep the whole request as JSON, similar to the API.